scripts/valid_payload.json

```diff
@@ -5,27 +5,19 @@
     "family_of_business": "Test Business Family",
```

Comment:
Yes, we should have one or more examples for any given transaction type (esp. like these SC provisioning transactions) and we can use them for documentation and even testing.

When we finalize, we should create a ticket for that.